packages/web/webpack.config.js

```diff
@@ -65,6 +65,7 @@ module.exports = merge(
 			"spdlog": path.join(vsFills, "spdlog.ts"),
 			"native-watchdog": path.join(vsFills, "native-watchdog.ts"),
 			"iconv-lite": path.join(vsFills, "iconv-lite.ts"),
+			"welcome": path.join(vsFills, "welcomePage.ts"),
```

Comment:
The alias will need to match the import, so in this case `vs/workbench/parts/welcome/page/electron-browser/vs_code_welcome_page`.